salt/states/win_lgpo.py

```diff
@@ -152,6 +152,20 @@ def _compare_policies(new_policy, current_policy):
             return False
 
 
+def _convert_to_unicode(data):
+    if isinstance(data, six.string_types):
+        data = data.replace('\x00', '')
```

Comment:
It's not clear exactly why this code is present. I know, but that's just because we had a conversation about it this last week 😂 

I'm sure I will forget in 6 months when I have to come back here to look 🙃 

Should we add a comment here about why we're doing the `.replace('\x00', '')`? Probably good to call out that if we correctly handle bytes vs. text that this functionality should become obsolete.

Reply:
Yeah, I meant to comment the code... but moved on before doing that... I'll add that

Reply:
Dun